pygmt/src/timestamp.py

```diff
@@ -62,13 +62,15 @@ def timestamp(
 
     Examples
     --------
-    >>> # Plot the GMT timestamp logo.
+    Plot the GMT timestamp logo.
+
     >>> import pygmt
     >>> fig = pygmt.Figure()
     >>> fig.timestamp()
     >>> fig.show()
 
-    >>> # Plot the GMT timestamp logo with a custom label.
+    Plot the GMT timestamp logo with a custom label.
+
     >>> fig = pygmt.Figure()
```

Comment:
I am wondering whether it is better to repeat `import pygmt` for the second example, as the two code examples are in two separate "cells"? When users copy and past the second example it will not work directly.

```suggestion
    Plot the GMT timestamp logo with a custom label.

    >>> import pygmt
    >>> fig = pygmt.Figure()
```

Reply:
I prefer not to repeat `import pygmt`. `import pygmt` initialize a new GMT session so it's time-consuming. Even new users should know that they need to `import pygmt` so it's not a big issue.

Reply:
I agree, it's only a smaller aspect and I am OK with leaving it as is.